docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -348,6 +348,7 @@ The following configuration set `Content-Type: application/vnd.android.package-a
 - `OFFLINE_MODE`: **false**: Disables use of CDN for static files and Gravatar for profile pictures.
 - `CERT_FILE`: **https/cert.pem**: Cert file path used for HTTPS. When chaining, the server certificate must come first, then intermediate CA certificates (if any). This is ignored if `ENABLE_ACME=true`. Paths are relative to `CUSTOM_PATH`.
 - `KEY_FILE`: **https/key.pem**: Key file path used for HTTPS. This is ignored if `ENABLE_ACME=true`. Paths are relative to `CUSTOM_PATH`.
+- `CA_FILE`: **<empty>**: CA file path used for TLS client certifiacte authentication. When used, enforces client certificate usage and prevents TLS tunnel otherwise. Paths are relative to `CUSTOM_PATH`.
```

Comment:
```suggestion
- `CA_FILE`: **<empty>**: CA file path used for TLS client certificate authentication. When used, enforces client certificate usage and prevents TLS tunnel otherwise. Paths are relative to `CUSTOM_PATH`.
```